keyboards/crkbd/keymaps/mkarikom/rules.mk

```
@@ -0,0 +1,14 @@
+
+# If you want to change the display of OLED, you need to change here
+SRC +=  ./lib/glcdfont.c \
+        ./lib/rgb_state_reader.c \
+        ./lib/layer_state_reader.c \
+        ./lib/logo_reader.c \
+        ./lib/keylogger.c \
+        # ./lib/mode_icon_reader.c \
+        # ./lib/host_led_state_reader.c \
+        # ./lib/timelogger.c \
+
+RGBLIGHT_ENABLE = no
+RGB_MATRIX_ENABLE = WS2812
```

```suggestion
RGB_MATRIX_ENABLE = yes
```

Driver is set separately now, because of ease of use issues.  So, yes/no for this, now, only.